lib/data/entity.js

```diff
@@ -172,7 +172,9 @@ const selectFields = (entity, properties, selectedProperties) => {
   const systemProperties = {
     createdAt: entity.createdAt,
     creatorId: entity.aux.creator.id.toString(),
-    creatorName: entity.aux.creator.displayName
+    creatorName: entity.aux.creator.displayName,
+    update: entity.aux.stats.update,
```

Comment:
Could we name this `updates` with an "s" to match `edits` in the submission OData?